### ISSUE TITLE
Save setting, when key has different case, causes throw of sql exception

### ISSUE THREAD

**Issue by chlupac** [oqtane/oqtane.framework#487]
example code:
```c#
        SettingService.SetSetting(settings, "casematters", _category);
        await SettingService.UpdateUserSettingsAsync(settings, PageState.User.UserId);
        SettingService.SetSetting(settings, "CaseMatters", _category);
        await SettingService.UpdateUserSettingsAsync(settings, PageState.User.UserId);
```
two keys from dictionary view, one from database view, (case insensitive collation)

**Comment by sbwalker** [comment]
Looks like a bug - keys must be treated as case insensitive in API.

**Comment by chlupac** [comment]
Yes it is, I will try to fix it.

**Comment by chlupac** [comment]
Solved in #491
